content/methods/discover/system-map.md

```diff
@@ -0,0 +1,50 @@
+---
+title: System map
+description: A diagram that shows how different factors and forces affect a given system. Unlike [journey maps](https://guides.18f.gov/methods/decide/journey-mapping/) or [service blueprints](https://guides.18f.gov/methods/decide/service-blueprint/), which visualize specific services.
+permalink: /methods/discover/system-map/
+redirect_from:
+  - system-map/
+layout: layouts/method-card
+tags: methods
+eleventyNavigation:
+  key: methods
+  parent: Discover
+  title: System map
+method:
+  title: Stakeholder and user interviews
+  what: A diagram that shows how different factors and forces affect a given system. Unlike [journey maps](https://guides.18f.gov/methods/decide/journey-mapping/) or [service blueprints](https://guides.18f.gov/methods/decide/service-blueprint/), which visualize specific services.
+  why: Mapping complex patterns can help us build consensus and identify opportunities. Maps can also be used to onboard new team members and inform product decisions.
+  timeRequired: 4-8 hours
+  category: discover
+  ---
```

Comment:
It looks like there's a space before the `---` that would mark the end of the frontmatter and the start of the content.

[This error](https://github.com/18F/guides/actions/runs/9652280246/job/26622055956#step:6:18) seems to be saying that L21 is still being interpreted as frontmatter.

I think if you delete the leading space on L19, it'll build.